src/main/java/io/github/phantamanta44/libnine/client/gui/component/GuiComponentManager.java

```diff
@@ -64,6 +84,10 @@ public boolean handleKeyTyped(char typed, int keyCode) {
             this.mouseOver = false;
         }
 
+        public GuiComponent getComp() {
```

Comment:
should be called `getComponent`

Reply:
It's auto-generated. Then comp should be called component, but you named it comp